projects/components/form/src/form.component.ts

```diff
@@ -76,7 +77,14 @@ export class PsFormComponent implements OnChanges, OnInit, OnDestroy {
     this._dataSource = value;
 
     if (this._dataSource) {
+      this.createErrorCardObserver();
       this._dataSourceSub = this._dataSource.connect().subscribe(() => {
+        if (!this._dataSource.exception) {
+          this._errorCardObserver.unobserve(this.errorCardWrapper.nativeElement);
+        } else if (this._dataSource.exception !== this._lastException) {

```

Comment:
If the exception changes from one to another, then you observe the wrapper twice.
You should rather check if you already observe the wrapper or not. (like what you did in the last commit)

Reply:
The IntersectionObserver seems to track the element instances which it observes. That means even if you try to observe the same element multiple times, the observer triggers only once per element instance.